### ISSUE TITLE
No FTP Support

### ISSUE THREAD

**Issue by cyberrufus** [lordmulder/INetGet#1]
![ftp](https://cloud.githubusercontent.com/assets/4472374/10825148/1564a4c4-7e3a-11e5-92f5-eaa15c81a9d1.jpg)

OS Windows 10 x64

**Comment by lordmulder** [comment]
Yes, as the error message says, FTP support is not implemented yet. It's on my "todo" list :sweat_smile:

**Comment by cyberrufus** [comment]
Thanks for the clarification. I was a bit confused because the documentation said it was supported.

**Comment by lordmulder** [comment]
Well, if you look at the [**changelog**](https://github.com/lordmulder/INetGet#version-100-2015-09-21), it says:

> FTP support is not yet implemented in this version.

:innocent:

**Comment by Zero3K** [comment]
When will FTP support be added?

**Comment by lordmulder** [comment]
No ETA, sorry 😓
